corehq/apps/formplayer_api/management/commands/clear_formplayer_dbs.py

```diff
@@ -0,0 +1,107 @@
+from corehq.apps.users.models import CouchUser
+from corehq.util.argparse_types import validate_integer
+from custom.covid.tasks import get_prime_restore_user_params, clear_formplayer_db_for_user
+
+
+class Command(BaseCommand):
+    help = inspect.cleandoc(
+        """Call the Formplayer clear-user-data API for users from CSV or matching criteria.
+        Usage:
+
+        Redirect stdout to file to allow viewing progress bar:
+        %(prog)s [args] > output.csv
+
+        ### With users in a CSV file ###
+
+        CSV Columns: "domain, username, as_user"
+
+        %(prog)s --from-csv path/to/users.csv
+
+        ### Query DB for users ###
+
+        %(prog)s --domains a b c --last-synced-days 2 --limit 1000
+
+        Use "--dry-run" and "--dry-run-count" to gauge impact of command.
+        """
+    )
+
+    def create_parser(self, *args, **kwargs):
+        # required to get nice output from `--help`
+        parser = super(Command, self).create_parser(*args, **kwargs)
+        parser.formatter_class = RawTextHelpFormatter
+        return parser
+
+    def add_arguments(self, parser):
+        parser.add_argument('--from-csv',
+                            help='Path to CSV file. Columns "domain, username, as_user". When this is supplied '
+                                 'only users in this file will be synced.')
+
+        parser.add_argument('--domains', nargs='+', help='Match users in these domains.')
+        parser.add_argument('--last-synced-hours', action=validate_integer(gt=0, lt=673), default=48,
+                            help='Match users who have synced within the given window. '
+                                 'Defaults to 48 hours. Max = 472 (4 week).')
+        parser.add_argument('--not-synced-hours', action=validate_integer(gt=0, lt=169),
+                            help='Exclude users who have synced within the given window. '
+                                 'Max = 168 (1 week).')
+
+        parser.add_argument('--limit', action=validate_integer(gt=0), help='Limit the number of users matched.')
+        parser.add_argument('--dry-run', action='store_true', help='Only print the list of users.')
+        parser.add_argument('--dry-run-count', action='store_true', help='Only print the count of matched users.')
+
+    def handle(self,
+               from_csv=None,
+               domains=None,
+               last_synced_hours=None,
+               not_synced_hours=None,
+               limit=None,
+               **options
+               ):
+        dry_run = options['dry_run']
+        dry_run_count = options['dry_run_count']
+
+        if from_csv:
+            users = _get_users_from_csv(from_csv)
+            if dry_run_count:
+                sys.stderr.write(f"\n{len(list(users))} users in CSV file '{from_csv}'\n")
+                return
+
+            for domain, request_user, as_username in users:
+                request_user_id = CouchUser.get_by_username(request_user).user_id
```

Comment:
Using `get_by_username` means both username and as_user need to be fully specified, right, like `user@domain.commcarehq.org`? Not sure how these CSVs will be generated or by who, but it might be helpful to use `normalize_username` so the suffix can be left off.